qualtran/simulation/classical_sim_test.py

```diff
@@ -168,6 +170,35 @@ def test_apply_classical_cbloq():
     np.testing.assert_array_equal(z, xarr)
 
 
+@pytest.mark.parametrize(
+    ['x', 'y', 'n_bits'],
+    [
+        (x, y, n_bits)
+        for n_bits in range(1, 5)
+        for x, y in itertools.product(range(1 << n_bits), repeat=2)
+    ],
+)
+def test_signed_addition_unsigned(x, y, n_bits):
+    assert signed_addition(x, y, 1 << n_bits, False) == (x + y) % (1 << n_bits)
+
+
+@pytest.mark.parametrize(
+    ['x', 'y', 'n_bits'],
+    [
+        (x, y, n_bits)
+        for n_bits in range(2, 5)
+        for x, y in itertools.product(range(-(2 ** (n_bits - 1)), 2 ** (n_bits - 1)), repeat=2)
+    ],
+)
+def test_signed_addition_signed(x, y, n_bits):
+    half_n = 1 << (n_bits - 1)
+    # Addition of signed ints `x` and `y`` is a cyclic rotation of the interval [-2^(n-1), 2^n) by `y`.
+    R = [*range(-(2 ** (n_bits - 1)), 2 ** (n_bits - 1))]
```

Comment:
is there a good reason why we're not using pep-8 variable names here?

Reply:
renamed